FWCore/MessageService/src/ELoutput.cc

```diff
@@ -95,7 +95,7 @@ namespace edm {
     // Useful function:
     // ----------------------------------------------------------------------
 
-    static ELstring formatTime(const time_t t) {  // Change log 7
+    static std::string formatTime(const time_t t) {  // Change log 7
```

Comment:
Should we 'modernize' this and put it into an anonymous namespace instead of declaring it `static`?

Reply:
I agree, I'll move to anonymous namespace (also elsewhere).